tools/validators/instance_validator/validate/generate_universe.py

```diff
@@ -59,10 +61,14 @@ def BuildUniverse(
         interactive=False)
     yaml_files = external_file_lib.RecursiveDirWalk(modified_types_filepath)
   else:
+    if default_types_filepath is None:
+      raise TypeError('default_types_filepath cannot be None! Should be '
+                      'specified somewhere')
```

Comment:
`in the arguments, please review the documentation for input?`or something instead of somehwere?